### ISSUE TITLE
Generate RoslynTools.MSBuild (xcopy-msbuild) RoslynTools.MSBuild.17.8.5

### ISSUE THREAD

**Issue by andriipatsula** [dotnet/arcade#14427]
Based on a [failure](https://dnceng.visualstudio.com/internal/_build/results?buildId=2365005&view=logs&j=b11b921d-8982-5bb3-754b-b114d42fd804&t=fb192a8b-e433-5fc8-e2b0-276ab015e7d5) reported on FR with

> Cannot find the file RoslynTools.MSBuild.17.8.3.nupkg in package 'RoslynTools.MSBuild 17.8.3' in feed 'dotnet-eng'

wiki: https://dev.azure.com/dnceng/internal/_wiki/wikis/DNCEng%20Services%20Wiki/1074/RoslynTools.MSBuild-(xcopy-msbuild)-generation

Generated package: https://dnceng.visualstudio.com/public/_artifacts/feed/dotnet-eng/NuGet/RoslynTools.MSBuild/overview/17.8.5

**Comment by riarenas** [comment]
@andriipatsula there's now a pipeline and script to generate this package as of https://github.com/dotnet/arcade/pull/14130. The pipeline is at: https://dnceng.visualstudio.com/internal/_build/results?buildId=2295858&view=results

It doesn't look like the linked instructions have been updated to account for that, but I would not follow those manual steps for this. 

I would try to use the pipeline to create the package. If you have any issues sync up with @chcosta

**Comment by andriipatsula** [comment]
Thanks @riarenas , I'll try to utilize the pipeline you posted. I have already published a package to our feed, but I think we can regenerate it.

**Comment by riarenas** [comment]
Oh, if we have a working package that's good too! I was hoping to save you some manual operations but it looks like you're already too far along for that.

**Comment by andriipatsula** [comment]
>Oh, if we have a working package that's good too

I am testing it right now. How it's "working" :)

**Comment by chcosta** [comment]
We should try the 1es package. I'll provide instructions later today

**Comment by chcosta** [comment]
I wouldn't block on me though

**Comment by andriipatsula** [comment]
The package is created and tested [here](https://dnceng.visualstudio.com/internal/_build/results?buildId=2365602&view=results). I am going to create a PR in the dotnet/arcade repo.

**Comment by andriipatsula** [comment]
PR to update the msbuild-xcopy in arcade: https://github.com/dotnet/arcade/pull/14442

**Comment by andriipatsula** [comment]
closing as resolved.

**Comment by chcosta** [comment]
@tkapin , You're right, I thought I had closed the loop here, but I didn't.  There IS a pipeline I created to handle the package generation, but at the same time I was doing that, I was having conversations with 1ES about producing a package for us. 1ES is now producing a package for us.  Hooray!  So the "work" involved going forward would (theoretically) be, take the package from their feed, publish it to our feed, and update the version number.  I'll prepare a PR to switch this over.  If the new package doesn't break the world, I'll update the documentation.
